myst_nbs/pymc3_howto/api_quickstart.myst.md

```diff
@@ -84,13 +87,16 @@ Every probabilistic program consists of observed and unobserved Random Variables
 help(pm.Normal)
 ```
 
-In the PyMC3 module, the structure for probability distributions looks like this:
+In the PyMC module, the structure for probability distributions looks like this:
 
-[pymc3.distributions](../api/distributions.rst)
-- [continuous](../api/distributions/continuous.rst)
-- [discrete](../api/distributions/discrete.rst)
-- [timeseries](../api/distributions/timeseries.rst)
-- [mixture](../api/distributions/mixture.rst)
+{ref}`_api_distributions`
```

Comment:
```suggestion
{ref}`pymc:api_distributions`
```

The rst syntax to define targets includes the _, so it is not part of the generated target. This is also referring to the versioned docs, not references in the example gallery. It is technically a different website so we should be explicit about what website we want it to point to (otherwise we might get the wrong link if multiple docs in intersphinx mapping like arviz or scipy also have a `api_distributions` target)

Reply:
Also, might be helpful: https://pymc-data-umbrella.xyz/en/latest/webinars/contributing_to_documentation/docs_presentation.html#cross-referencing-roles